splunk_add_on_ucc_framework/commands/build.py

```diff
@@ -508,6 +507,44 @@ def generate(
         logger.info(
             f"Installed add-on requirements into {ucc_lib_target} from {source}"
         )
+        if global_config.has_custom_search_commands():
+            for command in global_config.custom_search_commands:
+                file_path = os.path.join(source, "bin", command["fileName"])
+                if not os.path.isfile(file_path):
+                    logger.error(
+                        f"{command['fileName']} is not present in `{os.path.join(source, 'bin')}` directory. "
+                        "Please ensure the file exists."
+                    )
+                    sys.exit(1)
+
+                if (command["requireSeachAssistant"] is False) and (
+                    command.get("description")
+                    or command.get("usage")
+                    or command.get("syntax")
+                ):
```

Comment:
Can you please check is it possible to have this logic in the `schema.json`? Or would it be too complicated for the JSON schema to handle?

Reply:
Yes, it is possible to implement the above logic in `JSON schema` the reason I think we shouldn't go with `schema.json` approach is because, 

- `JSON Schema` validators typically reject invalid data, rather than logging a warning, and in this case I don't think we should stop the build process.
- `JSON Schema` library does not support custom error messages directly. It is better to validate and raise warning through above logic.